css/properties.json

```diff
@@ -9375,7 +9375,149 @@
     "computed": "asSpecified",
     "order": "perGrammar",
     "status": "standard",
-    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"  "stroke": {
```

Comment:
something looks off with the JSON here.. pulling the branch locally to have a look